pkg/server/router/operation.go

```diff
@@ -138,19 +144,29 @@ type ListOperationsResponse struct {
 //	@Tags			OperationAPI
 //	@Accept			json
 //	@Produce		json
-//	@Param			request	body		ListOperationsRequest	true	"request body"
+//	@Param			parent	query		string					false	"The name of the parent's resource. For example: `parent?=/presentation/submissions`"
+//	@Param			done	query		string					false	"A boolean value filtering for whether an operation is completed or not. For example: `?done=true`"
 //	@Success		200		{object}	ListOperationsResponse	"OK"
 //	@Failure		400		{string}	string					"Bad request"
 //	@Failure		500		{string}	string					"Internal server error"
 //	@Router			/v1/operations [get]
 func (o OperationRouter) ListOperations(c *gin.Context) {
-	var request ListOperationsRequest
-	invalidGetOperationsErr := "invalid list operations request"
-	if err := framework.Decode(c.Request, &request); err != nil {
-		framework.LoggingRespondErrWithMsg(c, err, invalidGetOperationsErr, http.StatusBadRequest)
-		return
+	parentParam := framework.GetParam(c, ParentParam)
+	doneParam := framework.GetParam(c, DoneParam)
+	var request listOperationsRequest
+	if parentParam != nil {
+		unescaped, err := url.QueryUnescape(*parentParam)
+		if err != nil {
+			errMsg := "failed un-escaping parent param"
+			framework.LoggingRespondErrWithMsg(c, err, errMsg, http.StatusInternalServerError)
+			return
+		}
+		request.Parent = unescaped
 	}
-
+	if doneParam != nil {
+		request.Filter = fmt.Sprintf("%s = %s", DoneIdentifier, *doneParam)
```

Comment:
if we are not using the filter as a dynamic thing someone can mess with on the front end, we should probably remove

Reply:
reworked